stock_split_picking/README.rst

```diff
@@ -52,6 +52,9 @@ Usage
 Bug Tracker
 ===========
 
+* When splitting a picking in an unassigned state, wizard won't
+  be auto completed with picking lines.
+
```

Comment:
These changes will be overridden automatically by a bot as the `README.rst` file is automatically generated.
It's ok to remove them, I don't think we need to document this.

Reply:
@sonhd91 Indeed, please put this in a supported section.

See https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Reply:
I will update it in USAGE.rst file and auto generate README again